laravel-app/app/View/Components/FollowButton.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace App\View\Components;
+
+use App\Models\User;
+use Closure;
+use Illuminate\Contracts\View\View;
+use Illuminate\View\Component;
+
+class FollowButton extends Component
+{
+    public User $user;
+    /** @var User $authUser */
```

Comment:
Is this still need?

Reply:
yes sir, both in local check and in blade file of follow button